fixtures/exalux/exoraa-zoom.json

```
@@ -0,0 +1,81 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Exoraa² Zoom",
+  "shortName": "Zoom",
+  "categories": ["Other", "Strobe"],
```

```suggestion
  "categories": ["Other"],
```
I don't see any strobe feature here, and the webpage doesn't mention a strobe.